_connect-files/api/documentation/api-quick-start.md

```diff
@@ -0,0 +1,282 @@
+
+        Either method will provide your application with an access token and a Stitch client ID. This information should be stored somewhere safe and passed into the header of every API request for the Stitch client's account. The token will never expire, but the user may revoke access at any time.
+
+        Keep in mind that Stitch client accounts are both owned and managed by the users themselves. For more information on authenticating with the API, refer to the [Authentication guide]({{ api.authentication }}).
+
+  - title: "Create a Destination"
+    anchor: "quick-start--create-a-destination"
+    content: |
+      If you're providing a destination for the Stitch client's account, we recommend connecting the destination immediately after the account is created. This ensures that Stitch will have a place to load replicated data as soon as data sources are added.
+
+      The first step to [creating a destination]({{ api.core-objects.destinations.create.anchor }}) is providing the attributes required for the destination's configuration, or form. These attributes are passed in the body of your request as the `connection` argument, along with the destination's `type`:
+
+      ```curl
+      curl -X POST {{ api.base-url }}{{ api.core-objects.destinations.create.name | flatify }}
+           -H "Authorization: Bearer <ACCESS_TOKEN>" 
+           -H "Content-Type: application/json"
+           -d "{
+                "type":"redshift",
+                "connection": {
+                  "host": "<HOST>",
+                  "port": 5439,
+                  "username": "<USERNAME>",
+                  "database": "<DATABASE>",
+                  "password": "<PASSWORD>",
+                  "ssl": false
+                  }
+               }"
+      ```
+
+      Refer to the [Destination Form Properties object]({{ api.form-properties.destination-forms.section | flatify }}) to retrieve the attributes required for the `connection` argument for each destination type. **Note**: Each destination has its own unique configuration and set of form attributes.
+
+  - title: "Create a Source"
+    anchor: "quick-start--create-a-source"
+    content: |
+      Source creation is performed through a sequence of [connection steps]({{ api.data-structures.connection-steps.section }}). The required steps and the order of those steps are unique to the source type and are defined in its [Report Card]({{ api.data-structures.report-cards.section }}) object. All source creation, however, begins at the `form` step.
+
+      We recommend using the [Source Type endpoint]({{ api.core-objects.source-types.section }}) to prep for source creation. This endpoint contains information about the configuration process and expected properties within each connection step for all source types. For example: You could use this endpoint to dynamically generate a UI or initial setup forms for each source type.
```

Comment:
This link `[Source Type endpoint]({{ api.core-objects.source-types.section }})` is broken -- I couldn't figure out how to fix it :-(

Reply:
I fixed this in the rewrite I just committed, so we should be good :)